ft/Styles

zbahati opened the pull request:
- Add progress bar in the component
- Add chapters part into components
- styles navbar , main and components
- add react progress bar by installing it using npm

[# Finishes ..]

Commit:
ft/Styles

- Add progress bar in the component
- Add chapters part into components
- styles navbar , main and components
- add react progress bar by installing it using npm

[# Finishes ..]

Ghiftee commented on src/index.css:
```diff
@@ -1,7 +1,276 @@
+  display: flex;
+}
+
+nav ul {
+  gap: 20px;
+}
+
+nav ul li a {
+  color: var(--gray-dark);
+}
+
+nav ul li .active {
+  color: var(--black-dark);
+  font-weight: 500;
+}
+
+nav button {
+  border-radius: 50%;
+  padding: 10px 12px;
+  border: none;
+}
```
- [ ] Kindly move the items labelled `1 and 2` to the position shown in the image. Also please ensure that the background of the part labelled `3` has the same background colour as in the [zeplin design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378)
- [ ] Also, kindly change the text `Home` to `Books` as in the given design and make the two nav items uppercase

### Zeplin design
![Screenshot 2023-06-21 at 16 08 38 (2)](https://github.com/zbahati/BookStore-React/assets/64774936/5e430224-e232-4429-9faf-221be5647958)

### Your design
![Screenshot 2023-06-21 at 15 42 05 (2)](https://github.com/zbahati/BookStore-React/assets/64774936/822f2ad7-90ca-43ee-a67c-d4ea08520718)

Ghiftee commented on src/index.css:
```diff
@@ -1,7 +1,276 @@
+  font-weight: 700;
+}
+
+.card-layout {
+  width: 100%;
+  flex-direction: row;
+  justify-content: space-between;
+}
```
- [ ] Kindly add the part labelled `4` to your design. It is the category of the book
- [ ] The parts labelled `5, 6 and 7` are the wrong colour in your design, kindly correct the colour as well as the font-family
- [ ] Kindly ensure that `8` is the correct colour and dimension as in the given design
- [ ] `9` is a different colour from the design, please correct it
- [ ] Your design is missing `10` please add it

### Given design
![Screenshot 2023-06-21 at 16 08 44 (2)](https://github.com/zbahati/BookStore-React/assets/64774936/3e9504a8-d20b-48e0-a588-66c350cf44b1)

### Your design
![Screenshot 2023-06-21 at 16 45 02 (2)](https://github.com/zbahati/BookStore-React/assets/64774936/f572b55c-af33-4adc-85e2-10b41e90c120)

Ghiftee commented on src/index.css:
```diff
@@ -1,7 +1,276 @@
+  margin: auto;
+}
+
+.newbook {
+  width: 85%;
+  margin: 20px auto;
+}
+
+.newbook h1 {
+  color: var(--gray-dark);
+  font-weight: 500;
+  padding: 10px 0;
+}
```
- [ ] The colour of the horizontal line labelled `11` is different from the one in your design, please correct it as it is important for developers to closely follow designs.
- [ ] Kindly correct `12` to `Category` as shown in the design

![Screenshot 2023-06-21 at 16 08 48 (2)](https://github.com/zbahati/BookStore-React/assets/64774936/d9995219-8378-4815-b6fe-9bd07ff1c263)

Ghiftee commented on src/index.css:
```diff
@@ -1,7 +1,276 @@
+h1 {
+  font-size: x-large;
+}
+
 body {
   font-family: 'Montserrat', sans-serif;
+  background-color: var(--pale-gray);
+  height: 100%;
+}
```
- [ ] Kindly go through your project and ensure that you are using all the correct font-family throughout
- [ ] Please ensure that your background color is the same as the given design

Ghiftee reviewed (changes requested):
## STATUS: CHANGES REQUIRED ♻️♻️
Hello @zbahati 👋🏽 

Good job working on the project so far! 
To highlight 👍🏽 
✔️ App runs smoothly
✔️ No linter errors

However, there are some issues that you need to work on to go to the next project but you are almost there!

![](https://user-images.githubusercontent.com/59707859/245511452-144e51b1-298e-4041-8d1e-8c012867348c.gif)

## Required Changes ♻️
_Check the comments under the review 👇🏽 _

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

Commit:
Ft/style

- Make changes regarding the requirement
- Remove input author
- Add select Category element
- styles all changes request

[# Finishes ...]

HAMMAS-SALEEM commented on src/components/CircularProgressbar.js:
```diff
@@ -1,17 +1,24 @@
+        value={percentage}
+        styles={buildStyles({
+          pathColor: `rgba(2, 144, 255, ${alpha})`,
+        })}
+      />
```
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

zbahati replied:
After the suggestion i create a logo and add it to the Read me

HAMMAS-SALEEM reviewed (approved):
# Status: Approved :green_circle: :1st_place_medal: :100:
![success](https://camo.githubusercontent.com/6ada59f6189497c1c2cae6d06f8154b7307a58643f232340e288140ed469eb28/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966)

### Hey @zbahati 👋

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Gitflow
✔️ Following Requirements
✔️ Your Design is really attractive 👍

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval. Still, we highly recommend to students that they should take these changes into consideration._

Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
Add my logo to the Readme
